### ISSUE TITLE
Create only one Smartdots Manual

### ISSUE THREAD

**Issue by kbekaert** [ices-eg/SmartDots#306]
Create only one SmartDots manual. Should be called "How to annotate" Manual.

**Comment by rellebo** [comment]
@AnnelieH , would you like us to use this issue to follow the work on the manual in attach which was already well advanced actually ;-) (Easier than sharePoint, right?)
I don't know the github user name for Maria...
[SmartDotsUserManualAgeAndMaturity_RE_MK.docx](https://github.com/ices-eg/SmartDots/files/10698042/SmartDotsUserManualAgeAndMaturity_RE_MK.docx)

**Comment by AnnelieH** [comment]
Didn't know so much was already done :-)
I think we keep the document on SP (so we don't end up with several versions of it) but yes we can communicate here. Is the document on SP now?

**Comment by AnnelieH** [comment]
@kbekaert @JCoadDavies do you know Maria Korta's GitHub name and can add her here? I can't find her in the list.

**Comment by JCoadDavies** [comment]
MKAlbisua is MKorta’s name and no i dont think the manual is on SP, best to work here

**Comment by AnnelieH** [comment]
Thanks @JCoadDavies. Ok then we continue here. @MKAlbisua just adding you here if you haven't seen the conversation before :-)

**Comment by AnnelieH** [comment]
I made a new version with the new (correct?) name "SmartDotsHowToAnnotateManual". It is on the SP - 2023 - working documents - software manual. I will start updating things in it and then of course check it out when I'm working. So you know if you want to work on it as well @rellebo and @MKAlbisua

**Comment by MKAlbisua** [comment]
Thank you Annellie. This morning I have taken a look at that new version with old name ( :-) ) because I remember that the maturity module screenshots were still to be updated (the final version of the module was not  published) ... Then I login in the smartdots module but as my event is finished I cannot do it, for example I cannot change the pictures status to have the screen shot of the traffic light, etc.. Should we ask Carlos to reopen the event?

**Comment by AnnelieH** [comment]
That is probably a good idea :-) I will focus on the age reading part since I have access to that type of event.

**Comment by rellebo** [comment]
hi @MKAlbisua & @AnnelieH, sorry for the delay on the subject (Closing of the sclero 2022 & IBTS for the data calls at the moment...)
I will take the opportunity if you wish to take screenshots and notes of the test version "SmartDots-Egg-Larvae-v4.0.4". To have a frame. I start well from the doc "smartDotsHowToAnnotateManual" on the SP 2023-->04. Working documents : Software Manual?

**Comment by rellebo** [comment]
@AnnelieH & @MKAlbisua ,  it appears that there is a general template consisting of a title, files/sample list, and pictures, with specific details depending on the modules (age/mat/eggs/larvae). Do you think it would be possible to start by standardizing a certain number of common descriptions, as was done for the event screen? Then, we could specify only the parts that are specific to each module. To make this work effectively, it would be necessary to further develop (@JCoadDavies or @kbekaert ) the process by harmonizing the labels for each module so that they could be easily incorporated into the document. I am willing to draft a preliminary version of this approach and see if it is feasible and readable. What do you think of this idea?

**Comment by kbekaert** [comment]
@rellebo, @AnnelieH , @MKAlbisua - the idea to create one manual for all modules was indeed to have one general part for all modules and specific parts for the 4 separate modules. In that way, when something changes in the general part, we don't have to update 4 manuals, but only one.  So I think it's a very good idea.

**Comment by AnnelieH** [comment]
@rellebo Yes if you could find a way to do that so it works well I think it is great. I will go out on a boat tomorrow and was planning to add all changes to the age reading events that was made during the last update (that I put in the newsletter) onboard. So I will do that if you don't have the file checked out.

**Comment by rellebo** [comment]
@AnnelieH, you will find here [the](https://filesender.renater.fr/?s=download&token=d443467b-d8e0-4a6d-ad4d-070d332de926) version that I started by sharing the globe elements there is still a lot of work on the content and form for sure I can continue on the whole if needed after your return, so from tomorrow you can work on the part age on board if you want and after i will continue.

**Comment by AnnelieH** [comment]
@rellebo Can't you put the file on SP? Then we can work on it when we have time. As long as it is not checked out. I will work now and then when I'm out but I might be out for 2 weeks and the deadline is mid-March so I don't think we can wait to finish to when I'm back.

**Comment by kbekaert** [comment]
Hi, seems like the training of the modules will take place half of May, so you might have some more time to finish the manual. But anyway, all that has been done must not be done any more. Thanks for the work!

**Comment by rellebo** [comment]
@AnnelieH If you want I devote time today and maybe start of thee next week to advance on, it that it is less draft and I deposit it at the end of the day on the SP,

**Comment by AnnelieH** [comment]
Thanks @rellebo I will start working on it tomorrow earliest.

**Comment by rellebo** [comment]
@AnnelieH @AnnelieH I stop for today there is still a lot of formatting and probably the resumption of my English but I think that the frame is emerging, I took the initiative not to follow all the changes because it became unreadable and very complex to follow, I have validated delete it still remains (the previous version on the sp if need), it should also continue the two new section I think mainly on the specifications egg and larvae.

 I can work again end of next week if necessary, I place the document here and on the SP as well as the images used
@AnnelieH good luck for the mission at sea ;-)

![larve](https://user-images.githubusercontent.com/119053715/222775979-a3f51f25-6b9e-4fb3-894d-581380985e20.jpg)
![mat](https://user-images.githubusercontent.com/119053715/222775985-722d9a92-5f89-4f68-8bfc-84b4f2e13e06.jpg)
<img width="1023" alt="oeufs" src="https://user-images.githubusercontent.com/119053715/222775988-73b7727f-b189-4d1a-a674-29eff8f7d686.png">
![oto](https://user-images.githubusercontent.com/119053715/222775994-d06ecda0-7f56-473f-ad13-3d0c8c05cd0a.jpg)
[SmartDotsHowToAnnotateManual_V1RE.docx](https://github.com/ices-eg/SmartDots/files/10884241/SmartDotsHowToAnnotateManual_V1RE.docx)

**Comment by rellebo** [comment]
Hi @AnnelieH  & @MKAlbisua, have you been able to start completing or rereading the document? I can work on it tomorrow if you wish.

**Comment by AnnelieH** [comment]
Hi @rellebo I'm really sorry but I haven't done anything yet (was impossible to get anything done on the boat and then completely swamped when I got back). Will start today. Got a message from Julie that it has to be done by March 31st.

**Comment by AnnelieH** [comment]
@rellebo @MKAlbisua I have worked on the manual today but we still have a lot to do! I started from the top and I think it is more or less ok until the end of page 13. I will continue with the common part and the age module, since that is the part I know best. Can you have a look at the maturity again? Do either of you have experience with the larvae and egg module? Deadline is next Wednesday 12/4.
